client/app/components/Calendar/index.js

```diff
@@ -124,20 +115,17 @@ class Calendar extends Component {
 
     render() {
         return (
-            <div className="wrapper-calendar">
                 <BigCalendar
                     selectable
-                    events={events}
+                    events={this.props.events}
                     popup
                     views={['month']}
                     messages={msg}
-                    components={getComponents(this.props)}
+                    components={this.props.getComponents(this.props)}
```

Comment:
If components are not passed by user, then default components needs to be used by Calendar.